### DISCUSSION TITLE
PDF references

### DISCUSSION THREAD

**Discussion by lab4here** [MuiseDestiny/zotero-reference#191] [General]
你好，

感谢您的插件。不幸的是，在Zotero 7（最新更新）以及您的插件的最新更新上，似乎都无法正常工作。此外，偏好设置窗口似乎也无法正常工作。

非常感谢。
我使用ChatGPT来进行翻译。

<img width="1800" alt="image" src="https://github.com/MuiseDestiny/zotero-reference/assets/118193011/84b29f3b-a12d-4de1-82fa-f6716433a96f">

**Comment by lab4here** [comment]
Excuse me, I just realized that it was necessary to install the "zotero-plugin-toolkit." Now, everything is working. Thank you!
However, I can't seem to get all the references... no problem.

不好意思，我刚刚意识到需要安装“zotero-plugin-toolkit”。现在一切都正常了。谢谢！
 然而，我似乎无法获得所有的参考文献……没关系。

**Comment by DongYX-01** [comment]
zotero7无法使用
